### ISSUE TITLE
Output is bugged when i do "to.json()"

### ISSUE THREAD

**Issue by KeparYTbcc** [browser-history/browser-history#204]
**Describe the bug**
basically i want to jsonify to "PrettyPrint" it, and it seems to not working

**To Reproduce**

```def chromehistory():
    entries = get_history().histories
    obj = generic.Outputs('history')
    obj.entries = entries
    data = json.loads(obj.to_json())
    mwssage = '====================================================\n'
    for item in data["history"]:
        mwssage += "Visited-date:", str(item["Timestamp"])+'\n'
        mwssage += "URL:", str(item["URL"])+'\n'
        mwssage += '====================================================\n'
    with open('outy\\History.txt', 'w+') as file:
        file.write(mwssage)
        print('History Saved to "outy" directory')
```
        
output:
`====================================================`
  
but entries:
`all my history`

**Comment by Samyak2** [comment]
This is definitely a bug, I can reproduce it. But it's easy to workaround it by simplifying it. `get_history()` returns an `Outputs` object already, you do not need to make a new one.

```python
entries = get_history()
data = json.loads(entries.to_json())
```

---

Also, you have a bug in this line: `mwssage += "Visited-date:", str(item["Timestamp"])+'\n'`. You have used a comma in place of a `+`.
Here's a working version of your code:
```python
entries = get_history()
data = json.loads(entries.to_json())
mwssage = '====================================================\n'
for item in data["history"]:
    mwssage += "Visited-date:" + str(item["Timestamp"]) + '\n'
    mwssage += "URL:" + str(item["URL"]) + '\n'
    mwssage += '====================================================\n'
with open('outy\\History.txt', 'w+') as file:
    file.write(mwssage)
    print('History Saved to "outy" directory')

```

**Comment by Samyak2** [comment]
Should be fixed by #266
